src/sagemaker/jumpstart/types.py

```diff
@@ -293,6 +293,7 @@ class JumpStartModelSpecs(JumpStartDataHolderType):
         "training_vulnerabilities",
         "deprecated",
         "metrics",
+        "hosting_prepacked_artifact_key",
```

Comment:
Would you be opposed to adding the `training_prepacked_script_key` in this PR along with similar logic for retrieval?

Reply:
Let's do that in another PR